features/custom_matchers/define_matcher.feature

```diff
@@ -335,3 +335,109 @@ Feature: define matcher
       | 4 examples, 2 failures               |
       | expected 9 to be a multiple of 2     |
       | expected 9 not to be a multiple of 3 |
+
+  Scenario: compare values
+
+    `RSpec` comes with some handy helper methods. One is `values_match?` which
+    is available if you define your matcher using the "matcher"-DSL.
```

Comment:
We actually start the documentation above, so I feel this should just be named appropriately, like: 

> Scenario: Using `values_match?` to compare values.